packages/markdown/component/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@astrojs/markdown",
+  "version": "0.1.0",
+  "type": "module",
+  "author": "withastro",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/withastro/astro.git",
+    "directory": "packages/markdown/component"
+  },
+  "bugs": "https://github.com/withastro/astro/issues",
+  "homepage": "https://astro.build",
+  "exports": {
+    ".": {
+      "astro": "./Markdown.astro"
+    }
```

Comment:
This works, but any reason not to use the normal `".": "./Markdown.astro"` here?

Reply:
Just seemed more correct, this package won't work outside of Astro, won't work in regular Node.js ESM, and the error message seemed more appropriate if you tried.